spec/requests/api/v1/live/submissions_swagger_spec.rb

```diff
@@ -0,0 +1,243 @@
+require "rails_helper"
+require "swagger_helper"
+
+RSpec.describe "GET submission", type: :request, swagger_doc: "v1/live/swagger.yaml" do
```

Comment:
TODO: could use some kind of shared example to call the exact same tests as for V1?!

Reply:
have used a shared example now